tests/Carbon/IssetTest.php

```diff
@@ -13,52 +13,92 @@
 
 namespace Tests\Carbon;
 
-use Carbon\Carbon;
 use Generator;
 use Tests\AbstractTestCase;
 
 class IssetTest extends AbstractTestCase
 {
-    public function testIssetReturnFalseForUnknownProperty()
+    public function testIssetReturnFalseForUnknownProperty(): void
     {
-        $this->assertFalse(isset(Carbon::create(1234, 5, 6, 7, 8, 9)->sdfsdfss));
```

Comment:
No need to create a new instance for all tests, when it's already available in `AbstractTestCase`